src/mpc-hc/mpc-hc.rc

```diff
@@ -1510,9 +1543,9 @@ BEGIN
         MENUITEM "&Filters",                    ID_FILTERS
         MENUITEM "S&haders",                    ID_SHADERS
         MENUITEM SEPARATOR
-        MENUITEM "&Audio",                      ID_AUDIOS
-        MENUITEM "Su&btitles",                  ID_SUBTITLES
-        MENUITEM "&Video Stream",               ID_VIDEO_STREAMS
+        MENUITEM "&Video Streams",              ID_VIDEO_STREAMS
+        MENUITEM "&Audio Streams",              ID_AUDIOS
+        MENUITEM "Su&btitles Streams",          ID_SUBTITLES

```

Comment:
It's unrelated to the PR cause and should not be included in my opinion.

As for changing this outside of PR... "Video Stream" is pretty much always disabled, other too were named like that since forever, I'm against. But I wouldn't really mind otherwise.